roles/ceph-defaults/defaults/main.yml

```diff
@@ -425,6 +425,10 @@ email_address: foo@bar.com
 # Monitor handler checks
 handler_health_mon_check_retries: 5
 handler_health_mon_check_delay: 10
+
+# Monitor protocols
+mon_host_v1_port: ":6789"
```

Comment:
we might have to remove the : from this parameter and move it up in lines 49/51/53 so that _port is just the port number